tokio/src/runtime/blocking/pool.rs

```diff
@@ -272,7 +284,7 @@ impl Spawner {
         shutdown_tx: shutdown::Sender,
         rt: &Handle,
         id: usize,
-    ) -> thread::JoinHandle<()> {
+    ) -> Result<thread::JoinHandle<()>, std::io::Error> {
```

Comment:
nit, take it or leave it: it might be a bit more concise to write
```suggestion
    ) -> std::io::Result<thread::JoinHandle<()>> {
```